signer/local_cosigner.go

```diff
@@ -393,6 +425,7 @@ func (cosigner *LocalCosigner) getEphemeralSecretPart(
 }
 
 // Store an ephemeral secret share part provided by another cosigner
+// Implements ThresholdEd25519Signature interface
 func (cosigner *LocalCosigner) setEphemeralSecretPart(req CosignerSetEphemeralSecretPartRequest) error {
```

Comment:
These should move out to a new file for the "local" threshold ed25519 signature implementation.